Docs/developer_guide/build_instructions/windows.md

```diff
@@ -56,20 +56,30 @@ Specify source, build, and Qt location and compiler version and start the build
 
 Release mode:
 
-```
+```bat
+cd C:\D
 mkdir C:\D\S4R
```

Comment:
We can also remove the explicit creation of the build directory

Reply:
Why would we remove the explicit creation of the build directory? We usually start a clean build by removing the entire build folder and it is useful that this script recreates it (otherwise CMake would dump error files etc. in the level above).

Reply:
Specifying `-B C:\D\S4R` takes care of creating the build directory.

Reply:
In that case, I think we can also remove these commands:
```
cd C:\D
```

Reply:
Assuming that the current drive doesn't need to be selected, removing `cd C:\D` would be sensible.

That said, I think keeping that command is a helpful hint indicating that everything will be taking place in that sub-directory.

Reply:
I removed only the `mkdir` command.